maquetado/scss/detail.scss

```diff
@@ -93,3 +95,74 @@
   font-size: 20px;
   line-height: 23px;
 }
+
+@media only screen and (max-width: 1024px) {
```

Comment:
Use a variable for 1024px; you can create other partial fiel for example _sizes.scss like _colors.scss